hasql.cabal

```diff
@@ -74,7 +74,7 @@ library
     -- parsing:
     attoparsec >= 0.10 && < 0.14,
     -- database:
-    postgresql-binary >= 0.9.1.1 && < 0.10,
+    postgresql-binary >= 0.10 && < 1,
```

Comment:
Okay, my mistake, I assumed the next major version would be 1.0. I will change it.